setup.cfg

```diff
@@ -0,0 +1,44 @@
+[metadata]
+name = quaternions
+version = 0.0.1
+author = David Boe
+# author_email = author@example.com
+description = A simple quaternion library
+license = MIT
```

Comment:
You can change the license info if you want.

I stubbed in the MIT license and also included the MIT license info in the LICENSE file.